config/routes.rb

```diff
@@ -4,6 +4,13 @@
   get '/users', to: 'users#index'
   get '/users/:id', to: 'users#show', as: :user
 
-  get '/users/:user_id/posts', to: 'posts#index', as: :user_posts
-  get '/users/:user_id/posts/:id', to: 'posts#show', as: :user_post
+  resources :users do
+    resources :posts do
+      resources :comments, only: [:create]
+    end
+  end
+
+  post '/users/:user_id/posts', to: 'posts#create', as: :create_post
+  post '/users/:user_id/posts/:post_id/comments', to: 'posts#create_comment', as: :create_comment
```

Comment:
- [ ] Great job with your implementation of the comments. However, per the project requirements, there should be a pair of routes in the `config/routes.rb` file for creating comments similar to what you did for `Posts create` and `Posts New`. Currently, you only have the `Post` route for comments in your project :+1: 
- Example
   ```ruby
      POST   /posts/:post_id/comments(.:format)     comments#create
    GET   /posts/:post_id/comments/new(.:format)     comments#new
   ```
- [ ] After doing this, you should ensure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments. :+1: